storage-stack-automation/README.md

```diff
@@ -0,0 +1,36 @@
+# Automation using Lambda to Deploy FSS- Storage Stack 
+
+Automated process to deploy FSS Storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.
+
+![](architecture.png)
+# Deploy via CloudFormation
+
+   * If not already present, [deploy a Scanner Stack](https://cloudone.trendmicro.com/docs/file-storage-security/stack-add/#AddScanner) in your Cloud One Account
+   * Obtain the Scanner stack's ID and SQS URL
+      - The Stack ID and SQSURL can be obtained with [list stacks api call](https://cloudone.trendmicro.com/docs/file-storage-security/api-reference/tag/Stack#operation/listStacks)
```

Comment:
If there is a way to get the Stack ID and SQSURL *not* using the API, please provide that as an alternative.

Reply:
Changed logic to take the name of scanner stack. No need for api call.
Can use the scanner stack outputs for name and url